front-end/src/variables.scss

```diff
@@ -1,31 +1,32 @@
-$main-bg:#F5F4F8;
-$config-bg: #e3e9f3;
-$text-config: #4c5a75;
-$pink-bg: #FF659E;
-$purple-bg: #584B82;
-$white: #ffffff;
+$configBg: #EAEAEA;
+$configText: #4c5a75;
+$configBtnColorOne:#FF659E;
+$configBtnColorTwo:#ffffff;
+$configLinkActiveBg: #584B82;
+$configLinkActiveTextColor: #ffffff;
 
-$EG-purple-dark: #5b205c;
-$EG-purple-light: #652e66;
-$EG-pink: #ed1261;
-$EG-blue: #00a0df;
-$EG-blue-dark: #4a326c;
-$EG-light-grey: #f2f2f2;
-$EG-white: #ffffff;
-$pink-text: #EF6298;
-$light-text: #DFDFFF;
-$medium-purple:#7573A3;
-$purple-text:#62568F;
-$dark-grey:#444444;
```

Comment:
I'd suggest to stick with color names, but use this service to name a color for convenience:
http://chir.ag/projects/name-that-color/#EF6298
No 'EG-' prefix though.
This article explains how different approaches turn out to be not optimal:
https://davidwalsh.name/sass-color-variables-dont-suck
I'm open to discussion.